tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -2297,12 +2297,14 @@ static Set<String> getPropertySet() {
   public static final String TEZ_MRREADER_CONFIG_UPDATE_PROPERTIES = "tez.mrreader.config.update.properties";
 
   /**
-   *  Frequency at which thread dump should be captured. Supports TimeUnits.
+   *  Frequency at which thread dump should be captured. Supports TimeUnits. This is effective only
+   *  when org.apache.tez.dag.app.ThreadDumpDAGHook is configured to tez.am.hooks or
+   *  org.apache.tez.runtime.task.ThreadDumpTaskAttemptHook is configured to tez.task.attempt.hooks.
    */
   @ConfigurationScope(Scope.DAG)
   @ConfigurationProperty
-  public static final String TEZ_THREAD_DUMP_INTERVAL = "tez.thread.dump.interval";
-  public static final String TEZ_THREAD_DUMP_INTERVAL_DEFAULT = "0ms";
+  public static final String TEZ_HOOK_THREAD_DUMP_INTERVAL = "tez.hook.thread.dump.interval";
```

Comment:
@okumin : I'm terribly sorry, I just realized that changing this causes more problems than benefits (changing config opts from one release to another), the class name also doesn't have "hook" in it, so it's fine to have this as "tez.thread.dump.interval", are you fine with chaging back?

Reply:
np. I renamed them back
https://github.com/apache/tez/pull/324/commits/293ce63e334e732c742cdc29ed6fe0fd9635d097